likecoin/admin/metabox.php

```diff
@@ -79,14 +79,30 @@ function likecoin_parse_publish_status( $publish_params ) {
  */
 function likecoin_parse_iscn_status( $publish_params ) {
 	global $post;
-	$post_id               = $post->ID;
-	$result                = array();
-	$iscn_hash             = $publish_params['iscn_hash'];
+	$post_id                = $post->ID;
+	$result                 = array();
+	$iscn_testnet_info      = get_post_meta( $post_id, LC_ISCN_DEV_INFO, true );
+	$iscn_mainnet_info      = get_post_meta( $post_id, LC_ISCN_INFO, true );
+	$iscn_hash              = $publish_params['iscn_hash'];
+	$iscn_data_api_endpoint = null;
+	$iscn_badge_endpoint    = null;
+	if ( $iscn_mainnet_info ) {
+		$iscn_data_api_endpoint = 'https://like.co/in/tx/iscn/'; // TODO: change to mainnet.
+		$iscn_badge_endpoint    = 'https://static.like.co/badge/iscn/'; // TODO: change to mainnet.
```

Comment:
sorry for confusion: a better name for the like.co/in url would be `iscn_view_page_url`
Also these would be the actual mainnet url, so no need for these comments